lib/internal/Magento/Framework/GraphQl/Query/ErrorHandler.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Framework\GraphQl\Query;
+
+/**
+ * Class ErrorHandler
+ *
+ * @package Magento\Framework\GraphQl\Query
+ */
+class ErrorHandler implements ErrorHandlerInterface
+{
+    /**
+     * @var \Magento\Framework\Logger\Monolog
+     */
+    private $clientLogger;
+
+    /**
+     * @var \Magento\Framework\Logger\Monolog
+     */
+    private $serverLogger;
+
+    /**
+     * @var array
+     */
+    private $clientErrorCategories;
+
+    /**
+     * @var array
+     */
+    private $serverErrorCategories;
+
+    /**
+     * @var \Magento\Framework\Logger\Monolog
+     */
+    private $generalLogger;
+
+    /**
+     * ErrorHandler constructor.
+     *
+     * @param \Magento\Framework\Logger\Monolog $clientLogger
```

Comment:
Please, fix a code style

Reply:
Please explain what do you mean: `@SuppressWarnings(PHPMD.LongVariable)` or, probably `ErrorHandler constructor.` comment?

I cannot see any warnings.

Reply:
@novikor - check static tests execution result:
https://testing-service.magento-community.engineering/reports/magento/graphql-ce/pull/355/b08a8e60-c5d9-47a9-9b68-17f76b08fa53/25714/Statics/allure-report-ce/index.html#categories/f7d7ae5f45206684c061cf24bc96e5c1/d2b318cecf601fd3/